libraries/ESP8266WiFiMesh/src/TcpIpMeshBackend.h

```diff
@@ -0,0 +1,283 @@
+/*
+  TcpIpMeshBackend
+ 
+  Copyright (c) 2015 Julian Fell and 2019 Anders Löfgren. All rights reserved.
+ 
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+// ESP-NOW is faster for small data payloads (up to a few kB, split over multiple messages). Transfer of up to 234 bytes takes 4 ms.
+// In general ESP-NOW transfer time can be approximated with the following function: transferTime = ceil(bytesToTransfer / 234.0)*3 ms.
+// If you only transfer 234 bytes at a time, this adds up to around 56kB/s. Finally a chance to relive the glory of the olden days
+// when people were restricted to V90 dial-up modems for internet access!
+// TCP-IP takes longer to connect (around 1000 ms), and an AP has to disconnect all connected stations in order to transfer data to another AP, 
```

Comment:
Why do all connected stations have to be dropped?

Reply:
That's the way the Arduino core works. Or at least it worked that way a year ago when the TCP/IP code was written. Quoting the cpp file: "Unlike WiFi.mode(WIFI_AP);, WiFi.mode(WIFI_AP_STA); allows us to stay connected to the AP we connected to in STA mode, at the same time as we can receive connections from other stations.  We cannot send data to the AP in STA_AP mode though, that requires STA mode. Switching to STA mode will disconnect all stations connected to the node AP (though they can request a reconnect even while we are in STA mode)."